metadetect/lsst/tests/test_lsst_metacal_exposures.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
```

Comment:
The test functions in this file could use more descriptive names.

Reply:
fixed